lib/elasticity/instance_constraints.rb

```diff
@@ -0,0 +1,7 @@
+module Elasticity
+
+  module InstanceConstraints
```

Comment:
I thought that CORE and MASTER will be defined here as well and referenced in other classes.